app/solutions.py

```diff
@@ -44,7 +32,7 @@ def check_solution(question_type, question_id, attempt):
         status = 200
 
     else:
-        attempt_response = AttemptError()
-        status = 404
+        raise HttpError(description="Question not found.",
```

Comment:
Nice
I guess you could extent this to be `f"Question {this.question_type} {this.question_id} not found"`